.github/workflows/main.yaml

```diff
@@ -1,73 +1,38 @@
-name: main
+name: Build
 
 on:
   push:
-    branches: [ new_ros2 ]
-  pull_request:
-    branches: [ new_ros2 ]
 
 jobs:
-  tii-deb-build:
+  build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        ros2_distro: [galactic]
     steps:
+      - uses: actions/checkout@v2
 
-      - name: Checkout rplidar_ros2
-        uses: actions/checkout@v2
-        with:
-          path: rplidar_ros2
-
-      # Run docker build
-      - name: Run rplidar_ros2 docker build
-        env:
-          ROS: 1
-          ROS_DISTRO: ${{ matrix.ros2_distro }}
-          PACKAGE_NAME: rplidar_ros2
-        run: |
-          set -eux
-          mkdir bin
-          pushd rplidar_ros2
-          ./build.sh ../bin/
-          popd
+      - uses: docker/setup-buildx-action@v1
 
-      - uses: jfrog/setup-jfrog-cli@v2
-        env:
-          JF_ARTIFACTORY_1: ${{ secrets.ARTIFACTORY_TOKEN }}
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/tiiuae/tii-rplidar
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=sha
+            type=raw,value=latest
```

Comment:
Looks good to me. Maybe we can to the endsWith changes later for all. I am pretty sure there would be another iteration of modifications anyway. 

```
type=raw,value=latest,enable=${{ endsWith(github.ref, github.event.repository.default_branch) }}
```

Reply:
Good catch! Yeah I think it's most efficient to do it on the larger batch